doc/howto/network_bridge_resolved.md

```diff
@@ -72,7 +72,70 @@ You must repeat the commands after each reboot and after LXD is restarted, or ma
 
 You can automate the `systemd-resolved` DNS configuration, so that it is applied on system start and takes effect when LXD creates the network interface.
 
-To do so, create a `systemd` unit file named `/etc/systemd/system/lxd-dns-<network_bridge>.service` with the following content:
+You can automate the systemd-resolved DNS configuration so that it is applied on system start and takes effect when LXD creates the network interface. To do so, there are two approaches you can take, detailed below. Use only one of the approaches.
+
+The first approach is recommended as more resilient; it is reapplied if the lxdbr0 network is re-created at any point. For example, updating and restarting LXD can occasionally cause the lxdbr0 network to be recreated.
+
+If you are unable to use the recommended approach for some reason, the second alternative approach can be used. However, if the lxdbr0 network is re-created, you must reapply that solution manually.
+
+### Recommended approach
+
+#### Create a `systemd` network file
+
+Create a `systemd` network file named `/etc/systemd/network/<network_bridge>.network` with the following content:
+
+```
+[Match]
+Name=<network_bridge>
+[Network]
+DNS=<dns_address>
+Domains=~<dns_domain>
+```
+
+So for example, `/etc/systemd/network/lxdbr0.network` with the following content (your IP, of course):
+
+```
+[Match]
+Name=lxdbr0
+[Network]
+DNS=10.167.146.1
+Domains=~lxd
+```
+
+#### Apply the updated configuration
+
+If you have rebooted since you first installed LXD, you only need to reload `systemd-resolved`:
+
+    systemctl restart systemd-resolved.service
+
+If you have _not_ rebooted your system since you first installed LXD, you must either:
+1. reboot the system, or 
+1. reload `systemd-networkd` (to reload the `.network` files) and restart `lxd` (to add the routing):
+
+    networkctl reload
+    snap restart lxd
+
+You can test that the updated configuration was applied by running:
+
+    resolvectl status
+
+The output should contain a section similar to the example shown below. You should see the configured DNS server and the `~lxd` domain:
+
+```
+[...]
+Link 4 (lxdbr0)
+    Current Scopes: DNS
+         Protocols: -DefaultRoute +LLMNR -mDNS -DNSOverTLS DNSSEC=no/unsupported
+Current DNS Server: 10.167.146.1
+       DNS Servers: 10.167.146.1
+        DNS Domain: ~lxd
+[...]
+```
+
+### Alternative approach
+
```

Comment:
````suggestion

```{warning}
This approach only automates applying your desired configuration when your system is rebooted. If LXD re-creates its bridge network outside of a system reboot, you must reapply the configuration manually with the following command:

    systemctl restart lxd-dns-<bridge_network>.service

Example:

    systemctl restart lxd-dns-lxdbr0.service
```

````